ui/src/app/views/app/component.html

```diff
@@ -1,7 +1,9 @@
 <ng-template *ngIf="(user | async).isSignedIn; then signedIn else notSignedIn"></ng-template>
 <ng-template #logoImages>
-  <img class="main-logo" src="assets/images/all-of-us-logo.svg" alt="All of Us"
+  <a routerLink="/" style="text-decoration: none">
```

Comment:
Thanks for adjusting things!

Can the stye rule go into CSS? I'm actually not quite sure why it's needed -- there isn't text in here, is there? But I often add a rule like `a img { border: none; }` to my global stylesheets.

Reply:
This style rule could totally go into CSS, doing that now. 
Its needed because it appears to be treating the image as text, and putting in an underline. Which means that since the image has some padding, it created a small artifact on the right side of the image, that this removed.